### PR TITLE
CMR-6848: Read CWIC from consortium for has_granules_or_cwic.

### PR DESCRIPTION
**Pull request by sxu123**

### CODE REVIEW

**Review by eereiter — APPROVED** [review]

File: search-app/docs/api.md [hunk 1]
```diff
@@ -1578,7 +1578,7 @@ Find collections that match all of the 'project' param values
 This supports `pattern`, `ignore_case` and option `and`.
 
 Find collections matching 'consortium' param value
-CWIC, FEDEO, CEOS, EOSDIS
```
Comment (jceaser) [inline search-app/docs/api.md:1581]: why remove example values, it's going to be hard enough to explain what a consortium is, having these names would help.

File: indexer-app/src/cmr/indexer/data/concepts/collection.clj [hunk 2]
```diff
@@ -211,11 +211,11 @@
     (let [free-and-open (:FreeAndOpenData use-constraints)]
       (case free-and-open
         true (distinct (conj consortiums "GEOSS"))
-        false (remove #(= "GEOSS" (str/upper-case %)) consortiums)
+        false (remove #(= "GEOSS" %) consortiums)
```
Comment (jceaser) [inline indexer-app/src/cmr/indexer/data/concepts/collection.clj:214]: why do we want to make any code case sensitive? I see the change on line 232, but do we ever expect to have a 'geoss' that is different from 'GEOSS'?

Comment (sxu123) [reply indexer-app/src/cmr/indexer/data/concepts/collection.clj:214]: It's not case sensitive. consortiums is already upper-cased in the index.

File: search-app/docs/api.md (hunk 1 above)
Comment (sxu123) [reply search-app/docs/api.md:1581]: This is just to fix the bug Matt mentioned in the documentation. Somehow the example in the line below it didn't show the escaped bracket, even though in the code the escape was there. so I just cleaned up the line and made it exactly the same as project example above.  I didn't really want to put these values there I forgot why I did. it's not complete and not directly describing the example below.

Comment (jceaser) [reply search-app/docs/api.md:1581]: I see there are some URLs with example values, that should work for my needs

**Review by jceaser — APPROVED** [review]

**Pull request merged**